apps/cli/app/services.py

```diff
@@ -1,5 +1,6 @@
 from beaupy import ValidationError, select
-from core import Actions, ContactPayload, ResponseType, controller
+from core import Actions, AllBirthdaysPayload, ContactPayload, ResponseType, controller
```

Comment:
This import will be updated, unfortunately

Reply:
Done